recipes/ctre/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,41 @@
+#include <ctre.hpp>
```

Comment:
not need to duplicate source code of test package. Please take a look at recipes templates in docs.